cmd/nginx-ingress/main_test.go

```diff
@@ -25,46 +25,68 @@ func TestValidateStatusPort(t *testing.T) {
 }
 
 func TestParseNginxStatusAllowCIDRs(t *testing.T) {
-
-	var tests = []struct {
+	var badCIDRs = []struct {
 		input         string
 		expected      []string
```

Comment:
no need to test for return value when `parseNginxStatusAllowCIDRs` returns an error